tests/controllers/test_sweep.py

```diff
@@ -55,3 +58,60 @@ def test_sweep_controller(monkeypatch):
             break
 
     assert sweep_controller.db.data["SweepConfig:a"]["stage"] == Stage.SUCCEEDED
+
+
+def test_sweep_controller_delete_sweep(monkeypatch):
+    sweep = Sweep(
+        sweep_id="a",
+        script_path=__file__,
+        total_experiments=2,
+        requirements=[],
+        parallel_experiments=1,
+        logger="tensorboard",
+        distributions={"best_model_score": Uniform(1, 10)},
+        framework="pytorch_lightning",
+        data={"a/": None},
+    )
+    sweep_controller = SweepController()
+    sweep_controller.db_url = "a"
+    monkeypatch.setattr(controller, "DatabaseClient", MockDatabaseClient)
+    config: SweepConfig = sweep.collect_model()
+
+    response = sweep_controller.run_sweep(config)
+    assert response == "Launched a Sweep 'a'."
+    assert len(sweep_controller.db.select_all(TensorboardConfig)) == 1
+
+    delete_config = DeleteSweepConfig()
+    delete_config.name = config.sweep_id
+
+    sweep_controller.delete_sweep(delete_config)
+    assert len(sweep_controller.db.select_all(TensorboardConfig)) == 0
+
+
+def test_sweep_controller_delete_experiment(monkeypatch):
```

Comment:
Let's add a test to delete an experiment from a sweep.

Reply:
Done